Assets/Scripts/PlayerInteraction/HeadsetCollide.cs

```diff
@@ -28,20 +26,23 @@ private void Start()
                     _LGrabber = _Grabbers[1];
                 }
             }
-            katydidEatSound = GetComponent<AudioSource>();
         }
+
         private void OnTriggerEnter(Collider other)
         {
-            if ((_Grabbers.Length > 0 && _RGrabber.GrabbedObject == other.gameObject) || (_Grabbers.Length > 1 && _LGrabber.GrabbedObject == other.gameObject))
+            // if the object being grabbed is the object that is being collided with
+            if ((_Grabbers.Length > 0 && _RGrabber.GrabbedObject == other.gameObject) || (_Grabbers.Length > 1 && _LGrabber.GrabbedObject == other.gameObject)) 
             {
                 if (other.gameObject.tag == "edible")
                 {
-                    katydidEatSound.Play();
-                    other.gameObject.GetComponent<Grabbable>().EndGrab();
-                    Destroy(other.gameObject);
                     onCollide.Invoke();
                 }
             }
```

Comment:
I think it should only invoke onCollide here. The UnityEvent should probably take in a parameter of Collider